src/templates/fr/2019/chapters/caching.html

```diff
@@ -433,15 +433,15 @@ <h2 id="comment-les-ttl-de-cache-se-comparent-ils-à-lâge-des-ressourcesnbsp"><
           </tr>
           <tr>
             <td>Bureau</td>
-            <td><p style="text-align: right;">70.7&nbsp;%</p></td>
-            <td><p style="text-align: right;">47.9&nbsp;%</p></td>
-            <td><p style="text-align: right;">59.2&nbsp;%</p></td>
+            <td><p class="numeric">70.7&nbsp;%</p></td>
+            <td><p class="numeric">47.9&nbsp;%</p></td>
+            <td><p class="numeric">59.2&nbsp;%</p></td>
           </tr>
           <tr>
             <td>Mobile</td>
-            <td><p style="text-align: right;">71.4&nbsp;%</p></td>
-            <td><p style="text-align: right;">46.8&nbsp;%</p></td>
-            <td><p style="text-align: right;">59.6&nbsp;%</p></td>
+            <td><p class="numeric">71.4&nbsp;%</p></td>
+            <td><p class="numeric">46.8&nbsp;%</p></td>
+            <td><p class="numeric">59.6&nbsp;%</p></td>
```

Comment:
```suggestion
            <td><p class="numeric">71.4%</p></td>
            <td><p class="numeric">46.8%</p></td>
            <td><p class="numeric">59.6%</p></td>
```

Reply:
This is intentional as noted in the [translators guide](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Translators'-Guide). In french they write "70 %" (with a space) while in English we use "70%" (without a space). We use a non-breaking space to prevent orphaning a number from the percent sign when they fall on a line break.